### PR TITLE
Improve notification action handling on Android

### PR DESCRIPTION
**Pull request by dluksza**
Improves integration with Android's notifications actions. Changes type of `action` property from `string` to `array` of `object`s. Also adds `id` property to each action, this way it splits action name (that can be internationalized) from the intent name.

Further more it moves `BroadcastReveiver` configuration to app's `AndroidManifest.xml` file. This fixes https://github.com/zo0r/react-native-push-notification/issues/626 where notification actions were not working (since there were no receiver registered) when app was not running.

Also updates README.md file and describes how to implement notification action support on android.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,35 +270,128 @@ For iOS, the repeating notification should land soon. It has already been merged
...
+  }
+}
+```
+
+### 5) Modify AndroidManifest.xml
+Add `service` and `receiver` with `intent-filter` to your `AndroidMainfest.xml` file.
+
+```
+<service android:name=".NotificationActionHandlerService" />
+<receiver android:name=".NotificatitonAcitonHandlerReceiver"
```
Comment (vishalmanohar) [inline README.md]: Typo: NotificatitonAcitonHandlerReceiver => NotificatitonActionHandlerReceiver

Comment (dluksza) [reply README.md]: This is fixed in followup commit

**Comment by dlimx** [comment]
Do you have any examples?

**Comment by likern** [comment]
@dluksza Hello! Could you shed light on status of this PR? Do you work on it?
It looks very appealing to get that functionality!

**Pull request closed without merging**